io.github.maurycyliebner.enve.metainfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.maurycyliebner.enve</id>
+  <launchable type="desktop-id">io.github.maurycyliebner.enve.desktop</launchable>
+  <name>enve</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <summary>2D animation</summary>
+  <description>
+    <p>
+      Flexible, user expandable 2D animation software for Linux and Windows.
+    </p>
+  </description>
+  <url type="homepage">https://maurycyliebner.github.io</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://user-images.githubusercontent.com/16670651/70745938-36e20900-1d25-11ea-9bdf-78d3fe402291.png</image>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release version="0.0.0" date="2019-09-10"/>
```

Comment:
I think it would be better to come up with something that `git describe` prints out or a number of commits? Either way project seems to rely on nightly builds rather than regular releases.

Reply:
I will open an issue upstream.

Reply:
upstream issue: https://github.com/MaurycyLiebner/enve/issues/122